src/RuleTypeConst.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace JSM.FluentValidation.AspNet.AsyncFilter
+{
+    /// <summary>
+    /// Const used do configure method WithRuleType.
+    /// </summary>
+    public static class RuleTypeConst
+    {
+        /// <summary>
+        /// Prefix used to override property name.
+        /// </summary>
+        public const string Prefix = "_RuleType";
```

Comment:
**question:** maybe I'm missing something here, but is this prefix really required?

Reply:
@luizhlelis this prefix is important to set correctly type in response.
We only set type, if exists an internal ErrorCode with this prefix.